unstructured/ingest/connector/discord.py

```diff
@@ -88,12 +89,19 @@ async def on_ready():
                 if self.days:
                     after_date = dt.datetime.utcnow() - dt.timedelta(days=self.days)
                 channel = bot.get_channel(int(self.channel))
+                if channel is None:
+                    raise FileNotFoundError("Channel not found.")
```

Comment:
do we know for a fact if this is None it's because the channel wasn't found versus a connectivity issue? asking because we hit the error earlier and it seemed to be an ephemeral issue, which would suggest the latter, right? I want to make sure we don't mislead the caller here.

Reply:
Got it, but then why were we seeing that in CI intermittently? ...because obviously that channel didn't disappear.

Reply:
also probably good to drop a line to that documentation as comment in the code here, but probably helpful for us to understand why this is the case intermittently in CI. seems like some _other_ case is also resulting in this being None (if None is what triggered the original error we were seeing)